File: apps/JobTrackerApi/Models/JobApplication.cs
namespace JobTrackerApi.Models
{
    public class JobApplication
    {
        public int Id { get; set; }
        public string? Company { get; set; }
        public string? Position { get; set; }
        public string? Status { get; set; }
        public DateTime AppliedDate { get; set; }
        public string? Notes { get; set; }
    }
}

File: apps/job-tracker-spring-api/src/main/java/com/example/jobtrackerapi/model/JobApplication.java
package com.example.jobtrackerapi.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

@Entity
public class JobApplication {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String company;
    private String position;
    private String status;
    private String appliedDate;
    private String notes;

    // Getters and Setters
}
